docs/index.html

```diff
@@ -21,13 +21,27 @@
                 <a class="desktop-v" href="#">Wander Gonz</a>
             </div>
             <div class="desktop-v">
-                <a class="desktop-v" href="#nav-bar">Portfolio</a>
+                <a class="desktop-v" href="#work-section">Portfolio</a>
                 <a class="desktop-v" href="#about-me">About</a>
                 <a class="desktop-v" href="#section-6">Contact</a>
                 <a class="desktop-v" href="#section-6"><img class="desktop-v" src="sprites/Union.svg" alt="message icon"></a>
             </div>
             <a href="#"><img src="sprites/coding-book.png" alt="Logo"></a>
-            <img src="sprites/Union.png" alt="">
+            <button class="menu-btn mobile" id="hamburguer" type="button"><img src="sprites/Union.png" alt="" ></button>
+            <div class="hidden menu-var style-menu" id="menu-v">
+               <div>
+                <button class="menu-btn menu-var" id="quitbtn" type="button"><img src="sprites/quitbtn.svg" alt=""></button>
+               </div>
+               <div class="menu-flex menu-var">
+                <a href="#work-section">Portfolio<img class="margin-left" src="sprites/greater-than.svg" alt=""></a>
+               </div>
+               <div class="menu-flex menu-var">
+                <a href="#about-me">About<img class="margin-left" src="sprites/greater-than.svg" alt=""></a>
+               </div>
+               <div class="menu-flex menu-var">
+                <a href="#section-6" >Contact<img class="margin-left" src="sprites/greater-than.svg" alt=""></a>
+               </div>
+            </div>
```

Comment:
- [ ] You did a great job on the functionality but you still need to work on the styles. For this project, you are supposed to stick to the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10)design as much as possible. Please use the correct font size for your mobile menu options, close(X) button and arrow icons  and make sure it matches the design on the template.

### Your design

![image](https://user-images.githubusercontent.com/83803414/161822200-7ad6237e-83d8-4932-8efe-c3bb389813f7.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/161822230-f579202e-212d-4758-a1e1-3d43f21766c3.png)